QuestionAnswering/QA_ICL/frontend/README.md

```diff
@@ -1,4 +1,28 @@
-# Frontend Development for Marie
+# Marie Frontend
+
+## Table of Contents
+- [Marie Frontend](#marie-frontend)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+  - [Project Structure](#project-structure)
+  - [Backend Integration](#backend-integration)
+
+
+## Overview
+
+The frontend of Marie serves as the user interface for interacting with the backend services that handle natural language queries related to chemistry data. Built using Next.js, the frontend communicates with the backend to perform operations such as querying chemical species and filtering zeolites, and it displays the results to users in a user-friendly manner.
+
+## Project Structure
+
+The frontend is organised into the following main directories:
+
```

Comment:
The frontend itself does NOT include the mock server. Please note that organisation of source code files and directory naming is not a 1-1 correspondence of the application architecture. It is entirely possible to have `backend`, `next_app_marie`, and `mock_backend` at the same directory level; it is purely my personal choice to group `next_app_marie` and `mock_backend` together in the same folder.